blog/2024-05-16-blog-status-of-lbse-in-webkit.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Status of LBSE in WebKit"
+author: rbuis
+permalink: /blog/status-of-lbse-in-webkit.html
+thumbnail: /assets/lbse-logo-wide.png
+preview: A look back at the last 7 months of development on LBSE in WebKit.
+---
+
+This blog entry gives an update on upstreaming and development of LBSE in WebKit in the last 7 months. For an explanation of
+what LBSE is and how it is related to WPE see this previous <a href="https://wpewebkit.org/blog/05-new-svg-engine.html">entry</a> as a refresher.
+
+[Describe wix and its sponsoring]
+
+<figure>
+  <a href="/assets/svg/wix-logo.svg" target="_blank"><img
+    style="dsplay: block; margin-left: auto; margin-right: auto; height: 60px;"
+    src="/assets/svg/wix-logo.svg"
+    alt="Diagram showing a box for the WPE backend in between the UI process and WPEWebProcess">
+  </a>
+</figure>
+
+## Project kick off and WebKit Contributors Meeting
+
+In **October 2023** we started the project, mostly by thinking of the design and roadmap. Also we did some general SVG bug fixing. First, visual overflow computation for SVG renderers was <a href="https://commits.webkit.org/268981@main">corrected</a>, which fixed quite some SVG pixel tests. Various visual bugs were also fixed like <a href="https://commits.webkit.org/269034@main">unnecessary repainting when viewBox is used on &lt;svg&gt; elements</a> and <a href="https://commits.webkit.org/269360@main">incorrect clipping for outermost &lt;svg&gt; elements</a>
+
+Also in the same month, we attended the <a href="https://docs.webkit.org/Other/Contributor%20Meetings/ContributorMeeting2023.html">WebKit Contributors</a> meeting, where we presented a talk on the LBSE (slides are <a href="https://teams.pages.igalia.com/webkit/contributors-meeting-presentations/2023/igalia-slides/integrating-the-new-LBSE.html">here</a>) and the feedback on LBSE at the meeting was very positive. Giving the talk early on actually helped us since we needed to have a good design in place.
```

Comment:
Generally we link to teams.pages.igalia do we?  Normally for talks we post and link them on slideshare I think

Reply:
Do you mean we generally _don’t_ link to teams.pages?

Reply:
Let me know. The reference itself is very useful I think to give the blog some more context.

Reply:
Yes, sorry typo - I indeed meant we _don't_ generally hotlink to those.  We should probably upload to slideshare (I would be surprised if it isn't already therE?) and link to that

Reply:
@nikolaszimmermann we still need a slideshare :)

Reply:
Please use the URL from https://www.slideshare.net/igalia/integrating-the-new-layerbased-svg-engine

Reply:
Done, thanks. D'oh, I think I saw that page before but forgot.